README.md

```diff
@@ -1,134 +1,144 @@
-# OTA Community Edition
+# OTA-Community-Edition
```

Comment:
This file's formatting is a bit weird. I think it could do with a once-over to improve that.

Reply:
Thanks for reviewing it , I cant really tell what weird is from your pov. If you could share something that would help clear my perspective, that would be awesome. Other than that , I will scrutinize it more and try to make it better.

Reply:
Well, this is how it looks to me:
![image](https://user-images.githubusercontent.com/5060433/181794823-b9c8715b-7768-42c0-8b22-d084d7dcb9b0.png)

I'd prefer if it were styled more consistently. Perhaps https://github.com/uptane/tuf-test-vectors/blob/master/README.md would be a good example. Most repos in the Uptane namespace have a similarly-formatted readme, although some are in asciidoc.